api/store/mock.go

```diff
@@ -41,25 +50,45 @@ func (db *MockDatabase) GetUser(email string) (*user.User, error) {
 }
 
 func (db *MockDatabase) PutProject(p *project.Project) error {
-	if _, ok := db.projects[p.ID]; ok {
+	if _, ok := db.projects[p.Name]; ok {
 		return errors.New("project already exists in the DB")
 	}
-	db.projects[p.ID] = p
+	db.projects[p.Name] = p
 	return nil
 }
 
 func (db *MockDatabase) GetProject(projectId string) (*project.Project, error) {
```

Comment:
kill this and the interface method corresponding to it is what i meant.

Reply:
kill GetProject??

Reply:
yes. or kill GetProjectByName they are the exact same thing

Reply:
yeah my bad didnt realize i left that in